lib/ledger-node-operations.js

```diff
@@ -7,6 +7,38 @@
 
 const async = require('async');
 const validator = require('./validator');
+const bedrock = require('bedrock');
+const BedrockError = bedrock.util.BedrockError;
+const {config: {constants}} = bedrock;
+const validate = require('bedrock-validation').validate;
+
+const validateContext = (context) => {
+  if(!context) {
+    throw new BedrockError('Operation must include a context.',
+      'NotFoundError');
+  }
+  if(!(typeof context === 'string' || Array.isArray(context))) {
+    throw new BedrockError('Operation context must be a string or an array.',
+      'TypeError');
+  }
+  if(typeof context === 'string' &&
+    context !== constants.WEB_LEDGER_CONTEXT_V1_URL) {
+    throw new BedrockError('Operation does not contain ' +
+      constants.WEB_LEDGER_CONTEXT_V1_URL, 'SyntaxError', {context});
+  }
+  if(Array.isArray(context) &&
+     context[0] !== constants.WEB_LEDGER_CONTEXT_V1_URL) {
+    throw new BedrockError('Operation must contain ' +
```

Comment:
Maybe something like this: "Operation context must contain " + ... + " as the first element." ...